src/main/java/org/pesmypetcare/webservice/entity/petmanager/PetEntity.java

```diff
@@ -164,15 +176,15 @@ public static void checkDateAndValueInteger(String key, Map<String, Object> body
      */
     public static void checkExercises(String key, Map<String, Object> body) {
         checkDateFormat(key);
-        if (body.size() != 4 || !body.containsKey("name") || !body.containsKey("description")
-            || !body.containsKey("endDateTime") || !body.containsKey("coordinates")) {
-            throw new IllegalArgumentException("Request body does not have a correct format");
+        if (body.size() != 4 || !body.containsKey(NAME) || !body.containsKey(DESCRIPTION)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean expression complexity is 4 (max allowed is 3).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5492924)